pom.xml

```diff
@@ -701,10 +701,11 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-compiler-plugin</artifactId>
-        <version>3.9.0</version>
+        <version>3.10.0</version>
         <configuration>
           <source>${java.version}</source>
           <target>${java.version}</target>
+          <createMissingPackageInfoClass>false</createMissingPackageInfoClass>
```

Comment:
New version brings a new feature which is not stable in some environments.
For some reason, the feature is enabled by default. We don't need this feature.